### ISSUE TITLE
JoinMidRound throws errors in console on people joining the server

### ISSUE THREAD

**Issue by LaFesta1749** [frankos6/SCPPlugins#44]
**JoinMidRound** throws errors in the console on people joining the server.

Exiled version: **Version 7.0.0-rc.2**
Plugin version: **1.3.1**

Console error message:

```
[2023-05-14 12:44:04.263 +02:00] [ERROR] [Exiled.API] Method "PlayerOnVoiceChatting" of the class "SCPPlugins.JoinMidRound.JoinMidRound" caused an exception when handling the event "Exiled.Events.Events+CustomEventHandler`1[[Exiled.Events.EventArgs.Player.VoiceChattingEventArgs, Exiled.Events, Version=7.0.0.0, Culture=neutral, PublicKeyToken=null]]"
[2023-05-14 12:44:04.275 +02:00] [ERROR] [Exiled.Events] System.ArgumentNullException: Value cannot be null.
                                 Parameter name: key
                                   at System.Collections.Generic.Dictionary`2[TKey,TValue].FindEntry (TKey key) [0x00008] in <05089c1b6989455fb1452c627044bac4>:0 
                                   at System.Collections.Generic.Dictionary`2[TKey,TValue].TryGetValue (TKey key, TValue& value) [0x00000] in <05089c1b6989455fb1452c627044bac4>:0 
                                   at SCPPlugins.JoinMidRound.JoinMidRound.PlayerOnVoiceChatting (Exiled.Events.EventArgs.Player.VoiceChattingEventArgs ev) [0x0000c] in <e8fd6bde8cf54b3d9ef987bc11d92218>:0 
                                   at Exiled.Events.Extensions.Event.InvokeSafely[T] (Exiled.Events.Events+CustomEventHandler`1[TInterface] ev, T arg) [0x00021] in <b38b71f059ac420c8cc4d863c480a890>:0
```

**Comment by LaFesta1749** [comment]
@frankos6 can you please take a look at this?

Thanks ♥

**Comment by frankos6** [comment]
This seems to happen when players use voicechat before fully loading, it doesn't affect how the plugin works
I will release a fix today

**Comment by frankos6** [comment]
Fixed in 6a34bc2aebb17c0f94ab4112a5c28e4f939e2fc3

**Comment by LaFesta1749** [comment]
> This seems to happen when players use voicechat before fully loading, it doesn't affect how the plugin works I will release a fix today

will there be an updated dll file?
